unit_testing/unitTests.py

```diff
@@ -3,22 +3,40 @@
 import argparse
 import subprocess
 import os
+import sys
+
+def run_tests_in_subdirectory(dir, regex, verbose):
```

Comment:
Neither regex nor verbose are used in this function. They should be appended to run_str